spatialos/UnrealGDK
# [UNR-5293] automatically start inspector process if not already started when PIE

#### Description
Previously, the inspector process would be killed when you closed the unreal editor. This meant that if you restarted the unreal editor, and started PIE then tried to use your previous open inspector browser tab, it would not work (because the inspector process was not open).

Now the inspector process is automatically started when PIE starts (so the same tab can be reused).

#### Release note
`Inspector process is automatically started when starting PIE. This means you can re-use existing inspector browser sessions.`

#### Tests
How did you test these changes prior to submitting this pull request?
I did a few tests to make sure it worked and to make sure it didn't break anything
Test 1 (functionality)
1. open ue editor
2. start play session
3. open inspector (2 and 3 can be swapped)
4. close ue editor

What automated tests are included in this PR?
none

STRONGLY SUGGESTED: How can this be verified by QA?
see tests section

#### Documentation
How is this documented (for example: release note, upgrade guide, feature page, in-code documentation)?
release note

#### Primary reviewers
If your change will take a long time to review, you can name at most two primary reviewers who are ultimately responsible for reviewing this request. @ mention them.
